lib/report.js

```diff
@@ -112,7 +100,17 @@ class ConciseReporter extends Reporter {
   listFailure(test, res, message) {
     if (res.stack) res.stack = renderStack(res.stack);
     this.printAssertErrorSeparator();
-    console.log(list(res, message, ['success']) + '\n');
+    let s = message;
+    for (const key of ['type', 'message', 'expected', 'actual', 'stack']) {
+      let value = res[key];
+      if (key === 'actual' || key === 'expected') {
+        let type = typeof value;
+        if (type === 'object') type = value.constructor.name;
+        value = `${JSON.stringify(value)} || ${type}`;
```

Comment:
I don't like using `'||'` to delimit value and type.
Using `':'` will probably be easier to understand.

Reply:
I just wanted to make a distinction between the value and type clear (i.e objects will contain colons as key-value separators). So now if you see || you may be sure that next comes the value type (unless obviously there is || in a string but that's much less possible).
Though I'm open to other separators.

Reply:
@lundibundi What do you think about using `is` as a separator, I think it will make it more human-readable,
e.g. `{} is Object`, `10 is number`, `'str' is string`?

Reply:
looks good, though not as distinct-looking as || but good enough. I hope it will be better when we add color support.